doc/source/index.rst

```diff
@@ -57,7 +57,7 @@ You can install PyAEDT on CPython 3.7 through 3.10 from PyPI with this command:
 
     pip install pyaedt
 
-To install PyAEDT with all extra packages (Matplotlib, NumPy, Pandas, and PyVista),
+To install PyAEDT with all extra packages (matplotlib, numpy, pandas, and pyvista),
```

Comment:
I'm OK with this change because it's referring to packages rather than organizations.